### PR TITLE
test: test 500 tables in a task

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
part of #1261 

### What is changed and how it works?
test 500 tables in a task (no sharding, 1 source). this case is added in tests/others_integration.txt now, we may find a better place after measuring CI

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: tests/many_tables/conf/diff_config.toml [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+user = "root"
+password = "123456"
+instance-id = "source-1"
+
+[[source-db]]
+host = "127.0.0.1"
+port = 3307
+user = "root"
+password = "123456"
+instance-id = "source-2"
```
Comment (csuzhangxc) [inline tests/many_tables/conf/diff_config.toml:34]: Can we remove these lines?

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Comment by lance6716** [comment]
replaying CI

File: tests/many_tables/conf/diff_config.toml (hunk 1 above)
Comment (lance6716) [reply tests/many_tables/conf/diff_config.toml]: done in https://github.com/pingcap/dm/pull/1301/commits/a8c66301229e028dbd952e76848338c05a8e96c9

**Comment by lance6716** [comment]
I'm changing CI now, could test it later

**Comment by lance6716** [comment]
changed `tests/run.sh` in https://github.com/pingcap/dm/pull/1301/commits/4f33b38e581c6a9eace7f32577f52ea1d8898c1a
now we could use `run_integration_test("dmctl_advance dmctl_basic dmctl_command")` in CI to run multiple cases

PTAL again

**Review by csuzhangxc — COMMENTED** [review]
LGTM

**Review by GMHDBJD — APPROVED** [review]
LGTM

**Pull request merged**